assets/src/edit-story/components/panels/elementAlignment.js

```diff
@@ -0,0 +1,366 @@
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as AlignBottom } from '../../icons/align_bottom.svg';
+import { ReactComponent as AlignTop } from '../../icons/align_top.svg';
+import { ReactComponent as AlignCenter } from '../../icons/align_center.svg';
+import { ReactComponent as AlignMiddle } from '../../icons/align_middle.svg';
+import { ReactComponent as AlignLeft } from '../../icons/align_left.svg';
+import { ReactComponent as AlignRight } from '../../icons/align_right.svg';
+import { ReactComponent as HorizontalDistribute } from '../../icons/horizontal_distribute.svg';
+import { ReactComponent as VerticalDistribute } from '../../icons/vertical_distribute.svg';
+import { dataPixels } from '../../units/dimensions';
+import getCommonValue from './utils/getCommonValue';
+import getBoundRect, {
+  calcRotatedObjectPositionAndSize,
+} from './utils/getBoundRect';
+
+const ElementRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10px 20px;
+  border-top: 1px solid ${({ theme }) => theme.colors.bg.v9};
+`;
+
+const IconButton = styled.button`
+  display: flex;
+  width: 28px;
+  height: 28px;
+  justify-content: center;
+  align-items: center;
+  background-color: unset;
+  cursor: pointer;
+  padding: 0;
+  border: 0;
+  border-radius: 4px;
+
+  &:hover {
+    background-color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.1)};
+  }
+
+  &:disabled {
+    pointer-events: none;
+    opacity: 0.2;
+  }
+
+  svg {
+    color: ${({ theme }) => theme.colors.mg.v2};
+    width: 28px;
+    height: 28px;
+  }
+`;
+
+const SeparateBorder = styled.div`
+  border-left: 1px dashed ${({ theme }) => rgba(theme.colors.bg.v0, 0.3)};
+  height: 12px;
+  margin-left: 4px;
+  margin-right: 4px;
+`;
+
+function ElementAlignmentPanel({ selectedElements, onSetProperties }) {
+  const boundRect = getBoundRect(selectedElements);
+  const isFill = getCommonValue(selectedElements, 'isFill');
+
+  const updatedSelectedElementsWithFrame = useMemo(
+    () =>
+      selectedElements.map((item) => {
+        const { id, x, y, width, height, rotationAngle } = item;
+        let frameX = x;
+        let frameY = y;
+        let frameWidth = width;
+        let frameHeight = height;
+        if (rotationAngle) {
+          const elementFrame = calcRotatedObjectPositionAndSize(
+            rotationAngle,
+            x,
+            y,
+            width,
+            height
+          );
+          frameX = elementFrame.x;
+          frameY = elementFrame.y;
+          frameWidth = elementFrame.width;
+          frameHeight = elementFrame.height;
+        }
+        return {
+          id,
+          x,
+          y,
+          width,
+          height,
+          frameX,
+          frameY,
+          frameWidth,
+          frameHeight,
+        };
+      }),
+    [selectedElements]
+  );
+
+  const isJustifyEnabled = isFill || selectedElements.length < 2;
```

Comment:
Shouldn't this be `isJustifyDisabled`? 🤔

Reply:
Yes, absolutely Disabled 👍